tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/GraphQL/HotChocolate/ExecuteAsyncIntegration.cs

```diff
@@ -0,0 +1,82 @@
+// <copyright file="ExecuteAsyncIntegration.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.ComponentModel;
+using System.Linq;
+using System.Threading;
+using Datadog.Trace.ClrProfiler.CallTarget;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.GraphQL.HotChocolate
+{
+    /// <summary>
+    /// GraphQL.Execution.ExecutionStrategy calltarget instrumentation
```

Comment:
I guess is not the right comment for this integration

Reply:
True 🙈